File: Back-end/FITExamAPI/FITExamAPI/Models/Result.cs
namespace FITExamAPI.Models
{
    public class Result
    {
        public int Id { get; set; }
        public int? UserId { get; set; }
        public int? ExamId { get; set; }
        public int? TotalCorrectAnswer { get; set; }
        public double? Score { get; set; }
        public DateTime? ExamDate { get; set; }
        public int? OverallTime { get; set; }
        public virtual User? User { get; set; }
        public virtual Exam? Exam { get; set; }
        public virtual ICollection<ResultDetail>? ResultDetails { get; set; } = new List<ResultDetail>();
    }
}

File: Front-end/FIT_Exam/app/src/main/java/vn/edu/hcmuaf/fit/fit_exam/model/Result.java
package vn.edu.hcmuaf.fit.fit_exam.model;

import java.io.Serializable;

public class Result implements Serializable {
    private int id;
    private int userId;
    private int examId;
    private int totalCorrectAnswer;
    private double score;
    private String startTime;
    private String endTime;
    private User user;
    private Exam exam;

    public Result() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public int getExamId() {
        return examId;
    }

    public void setExamId(int examId) {
        this.examId = examId;
    }

    public int getTotalCorrectAnswer() {
        return totalCorrectAnswer;
    }

    public void setTotalCorrectAnswer(int totalCorrectAnswer) {
        this.totalCorrectAnswer = totalCorrectAnswer;
    }

    public double getScore() {
        return score;
    }

    public void setScore(double score) {
        this.score = score;
    }

    public String getStartTime() {
        return startTime;
    }

    public void setStartTime(String startTime) {
        this.startTime = startTime;
    }

    public String getEndTime() {
        return endTime;
    }

    public void setEndTime(String endTime) {
        this.endTime = endTime;
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }

    public Exam getExam() {
        return exam;
    }

    public void setExam(Exam exam) {
        this.exam = exam;
    }
}
